bookwyrm/setup.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# Perform initial setup so that the users don't have to
+# This should only be done in the web container;
+# the celery and flower containers should wait for the canary to exist
+
+CANARY=/app/static/.dbinit_done
+
+info() { echo >&2 "$*" ; }
+die() { echo >&2 "$*" ; exit 1 ; }
+
+if [ "$1" == "exit" ]; then
+	info "**** base container exiting"
+	exit 0
+fi
+
+if [ -z "$DB_INIT" ]; then
+	while [ ! -r "$CANARY" ]; do
+		info "**** Waiting for database and migrations to finish"
+		sleep 10
```

Comment:
Personally I would:

```suggestion
		sleep 0.5
```

since the test is very cheap.